operator/src/main/resources/instances/standard.properties

```diff
@@ -13,6 +13,10 @@ zookeeper.jvm-xms=1G
 zookeeper.container-cpu=1000m
 zookeeper.container-memory=4Gi
 
+cruisecontrol.container-memory=2Gi
```

Comment:
Between this and the 4.10 resource reduction (there's 336Mi less allocatable per node) we'll make an adjustment to the kafka.container-memory.  Assuming that we want to leave 1Gi of padding, the new number would be 17.3Gi instead of 18.1Gi.

With the cpu this low, I don't believe that we need to make any adjustment to the broker cpu.

Reply:
@rareddy @katheris @k-wall This is assuming that cruise control could be enabled even with the 3 brokers.  If it will always remain 6 or more, then we don't need to make any resource changes to account for cruise control.

Reply:
I suspect we eventually come around to the model where CC will be turned on with the 3 brokers, so maybe IMO it is better to align to that to begin with.